src/plugins/statusPage/public/statusPageMetric.js

```diff
@@ -54,12 +52,18 @@ require('ui/modules')
           break;
       }
 
-      self.chartOptions = getChartOptions(self.numberType);
-
       $scope.$watch('data', function (data) {
         self.rawData = data;
         self.chartData = readStatData(self.rawData, self.seriesNames);
         self.averages = calcAvg(self.chartData, self.numberType);
+
+        var unit = '';
+        self.averages = self.averages.map(function (average) {
+          var parts = average.split(' ');
+          unit = parts[1] ? ' (' + parts[1] + ')' : '';
+          return parts[0];
+        });
+        self.extendedTitle = self.title + unit;
       });
```

Comment:
Why use the average here? Why not just use the last value? Also, is this still polling and storing history somewhere? Can we get rid of that?